hrms/hr/doctype/employee_advance/employee_advance.json

```diff
@@ -187,6 +188,7 @@
    "fieldname": "pending_amount",
    "fieldtype": "Currency",
    "label": "Pending Amount",
+   "no_copy": 1,
```

Comment:
update `modified` timestamp in the doctype else these changes won't sync on sites on migrate